jib-core/src/main/java/com/google/cloud/tools/jib/configuration/DockerHealthCheck.java

```diff
@@ -211,4 +211,15 @@ private DockerHealthCheck(
   public Optional<Integer> getRetries() {
     return Optional.ofNullable(retries);
   }
+
+  /**
+   * Returns {@code true} if the entire health check is inherited from the base image, {@code false}
+   * if not.
+   *
+   * @return {@code true} if the entire health check is inherited from the base image, {@code false}
+   *     if not
+   */
+  public boolean hasCommand() {
+    return command != null && !command.isEmpty();
```

Comment:
I think a `DockerHealthcheck` not having a command should not be a valid `DockerHealthcheck`. For example, if someone decided to create a `DockerHealthcheck` with no command but other values set, rather than not doing anything (which would be different from what the user expected), it should error. I'm thinking that defining an inherited healthcheck shouldn't be a healthcheck with no command, but rather just not having a healthcheck (`null`), since in a `Dockerfile`, `HEALTHCHECK` can't be defined with no command.

Reply:
I personally feel like we should avoid using a `null` healthcheck instead of an explicit way of configuring whether or not to inherit, since it's a bit ambiguous as to whether or not `null` means to inherit or disable healthcheck. Will update the proposal doc or make a new one to get team consensus about what to do here.